tools/download_counters/README.md

```diff
@@ -0,0 +1,44 @@
+# Download Counters Script
+
+This script retrieves the download counts of a list of repositories from the GitHub API.
+
+## Prerequisites
+
+- Bash
+- `curl`
+- `jq` (for parsing JSON responses)
+
+## Setup
+
+1. Install `jq` if you don't have it already:
+
+   ```bash
+   sudo apt-get install jq
+   ```
+
+2. Obtain a GitHub personal access token. You can create one [here](https://github.com/settings/tokens).
+
+## Usage
+
+1. Clone the repository or download the script.
+
+2. Open a terminal and navigate to the directory containing the script.
+
+3. Set the `GITHUB_TOKEN` environment variable with your GitHub personal access token:
+
+   ```bash
+   export GITHUB_TOKEN=your_github_token
```

Comment:
for me, it's less system intrusive if you manage the github_token as an input parameter of the script instead of using a new system variable which can potentially be used for other usages in local machine